/**:
    ros__parameters:
        feature_extract_enable: false
        point_filter_num: 3
        max_iteration: 3
        filter_size_surf: 0.5
        filter_size_map: 0.5
        cube_side_length: 1000.0
        runtime_pos_log_enable: false
        map_file_path: "/home/z/rm_sim/src/rm_simulation/FAST_LIO/PCD/test.pcd"

        common:
            lid_topic:  "/livox/lidar/pointcloud"
            imu_topic:  "/livox/imu"
            time_sync_en: false         # ONLY turn on when external time synchronization is really not possible
            time_offset_lidar_to_imu: 0.0 # Time offset between lidar and IMU calibrated by other algorithms, e.g. LI-Init (can be found in README).
                                        # This param will take effect no matter what time_sync_en is. So if the time offset is not known exactly, please set as 0.0

        preprocess:
            lidar_type: 1                # 1 for Livox serials LiDAR (MID360), 2 for Velodyne LiDAR, 3 for ouster LiDAR, 4 for any other pointcloud input
            scan_line:  4
            blind: 0.5
            timestamp_unit: 3
            scan_rate: 10

        mapping:
            acc_cov: 0.1
            gyr_cov: 0.1
            b_acc_cov: 0.0001
            b_gyr_cov: 0.0001
            fov_degree:    360.0
            det_range:     100.0
            extrinsic_est_en:  false     # false: use fixed extrinsic for localization (recommended for better stability)
            extrinsic_T: [ -0.011, -0.02329, 0.04412 ]
            extrinsic_R: [ 1., 0., 0.,
                            0., 1., 0.,
                            0., 0., 1.]

        publish:
            path_en: true                # true: publish Path
            effect_map_en: false         # true: publish Effects
            map_en: false                # false: disable local map publishing to save bandwidth (use global map instead)
            scan_publish_en:  true       # true: publish registered scan for localization
            dense_publish_en: false      # false: low down the points number in a global-frame point clouds scan.
            scan_bodyframe_pub_en: true  # true: output the point cloud scans in IMU-body-frame

        pcd_save:
            pcd_save_en: false           # false: disable map saving in localization mode
            interval: -1                 # how many LiDAR frames saved in each pcd file;
                                        # -1 : all frames will be saved in ONE pcd file, may lead to memory crash when having too much frames.
